phaser-react/src/gameplay.js

```diff
@@ -29,21 +30,14 @@ export const gameplayMachine = createMachine({
         catalyst_country: {
             on: {
                 CONTINUE: {
-                    target: "catalyst_objective",
+                    target: "objective",
                 },
             },
         },
-        catalyst_objective: {
-            on: {
-                CONTINUE: {
-                    target: "decision_AB",
-                },
-            },
-        },
-        decision_AB: {
+        objective: {
             on: {
                 ILLEGAL_OPTION_SELECTED: {
-                    target: "decision_A1",
+                    target: "find_coyote",
                     // cond: "isSelectedIllegalOption",
```

Comment:
Would you want to add a TODO here on top of the commented sections of code to indicate to the reviewer that you'd want to revisit adding this in (`cond`, etc.)?

Reply:
would the guards be necessary here to control the state transition if for this state's component file, i just created 2 buttons that takes the player to the appropriate next state?

Reply:
hmm good question - I'm not entirely sure! but worth it for you to try!